src/backend/profile/profile.service.ts

```diff
@@ -154,7 +155,13 @@ export async function updateUserData(
     },
     data: userDataToUpdate,
   })
-
+  if (userDataToUpdate?.state) {
+    // I have no idea how to get current app url for link
+    discordNotification(
+      `User's ${updatedUser.fullName} profile has got new status: ${userDataToUpdate.state}! Profile`,
+      `http://localhost:3000/dashboard/profile/${updatedUser.userId}`,
```

Comment:
**suggestion:** You can do this by putting origin env, so `${process.env.NEXT_PUBLIC_APP_ORIGIN_URL}/dashboard/profile/${updatedUser.userId}`

Reply:
Env var it is then, great, thanks for help!